modules/flowable-cmmn-converter/src/test/java/org/flowable/test/cmmn/converter/ExitCriteriaBlockingExpressionCmmnXmlConverterTest.java

```diff
@@ -63,30 +63,29 @@ public void validateModel(CmmnModel cmmnModel) {
         assertThat(planItemTaskA).isNotNull();
         assertThat(planItemTaskA.getDefinitionRef()).isEqualTo("taskDefinition");
 
-
         PlanItem planItemTaskB = cmmnModel.findPlanItem("taskB");
         assertThat(planItemTaskB).isNotNull();
         assertThat(planItemTaskB.getDefinitionRef()).isEqualTo("taskDefinition");
         assertThat(planItemTaskB.getExitCriteria())
-                .extracting(Criterion::getId, Criterion::getSentryRef)
```

Comment:
Only whitespaces changes. We adapted the code style for IntelliJ to have one more level of nesting in order to align with the entire team. Are you using Eclipse or IntelliJ?

Reply:
@filiphr I am using IntelliJ.

Reply:
I've imported the style configuration and replaced the file.